src/main/resources/db/changelog/changeset/create/create-tables.yaml

```diff
@@ -0,0 +1,174 @@
+databaseChangeLog:
+  - changeSet:
```

Comment:
I recommend to write id of changeset in the following way: id: changeset-1.0.0" If it's required to define multiple changesets in one changelog, then you can follow this pattern:
```
id: changeset-1.0.0_001 - first changeset
id: changeset-1.0.0_002 - second changeset
...
```

Reply:
and each table creation should be in individual changeset, as some databases doesn't support transactions for DDL operations